Audible-Altimeter/src/runner.cpp

```diff
@@ -22,20 +22,39 @@ struct AltitudeToAudioSample {
   AUDIO_SAMPLE_ID sample_id{};
 };
 
-constexpr std::array<AltitudeToAudioSample, 8> k_altitudes{
+constexpr std::array<AltitudeToAudioSample, 29> k_altitudes{
     AltitudeToAudioSample(200, AUDIO_SAMPLE_ID::TWO),
     AltitudeToAudioSample(300, AUDIO_SAMPLE_ID::THREE),
     AltitudeToAudioSample(400, AUDIO_SAMPLE_ID::FOUR),
-    AltitudeToAudioSample(1'000, AUDIO_SAMPLE_ID::ONE),
+    AltitudeToAudioSample(500, AUDIO_SAMPLE_ID::FIVE),
+    AltitudeToAudioSample(600, AUDIO_SAMPLE_ID::SIX),
+    AltitudeToAudioSample(700, AUDIO_SAMPLE_ID::SEVEN),
+    AltitudeToAudioSample(800, AUDIO_SAMPLE_ID::EIGHT),
+    AltitudeToAudioSample(900, AUDIO_SAMPLE_ID::NINE),
+    AltitudeToAudioSample(1'000, AUDIO_SAMPLE_ID::TEN),
```

Comment:
are we just stubbing the ones we don't have yet?

Reply:
I believe these are all of the audio files we would want below 15,000 feet. I recycled some around here mainly because that is what VOG did: https://vog.ee/

For a little bit of context, 1,000 feet is about when we start our landing pattern. This is also when the altimeter begins speaking in "tens" instead of "thousands", with the exception of 1,500, which is also spoken as "hundreds"